pyls/python_ls.py

```diff
@@ -35,15 +36,17 @@ def handle(self):
         self.delegate.start()
 
 
-def start_tcp_lang_server(bind_addr, port, handler_class):
+def start_tcp_lang_server(bind_addr, port, check_parent_process,
+                          handler_class):
```

Comment:
Let's leave this in a single line:

    def start_tcp_lang_server(bind_addr, port, check_parent_process, handler_class):